src/components/Member.svelte

```diff
@@ -0,0 +1,32 @@
+<script>
+  export let member = {};
+  let { imageSrc, name, bio, socials = [], website = {} } = member;
+</script>
+
+<div class="max-w-sm rounded overflow-hidden shadow-xl">
+  <div class="h-64 flex-none bg-center bg-cover rounded-t lg:rounded-t-none lg:rounded-l text-center overflow-hidden" style="background-image: url('/members/{imageSrc}')"/>
+  <div class="px-6 py-4">
+    <div class="font-bold text-xl mb-2">
+      {name}
+    </div>
+    <p class="text-gray-700 text-base">
+      {bio}
+    </p>
+  </div>
+  <div class="px-6 py-4 flex justify-between">
+    {#if socials}
+      <div class="flex justify-start">
+        {#each socials as social}
+          <a href="{social.link}" aria-label="View {name} on {social.key}">
+            <i aria-hidden="true" class="fa fa-{social.key} mr-4"></i>
+          </a>
+        {/each}
+      </div>
+    {/if}
+    {#if Object.keys(website).length}
```

Comment:
Since `website` is an object, we could just check for existence here: `#if website`

Reply:
this hasn't worked actually. I was a bit confused but haven't thought about it. It prints "undefined" if I check if website

Reply:
maybe then check `#if website !== undefined`?
Checking if website has keys is somewhat missleading if you read that template

Reply:
I was wondering why website is undefined at all. on top of this file I assign an empty object to website